openmetadata-ui/src/main/resources/ui/src/components/Webhooks/WebhooksV1.tsx

```diff
@@ -128,83 +127,80 @@ const WebhooksV1: FC<WebhooksV1Props> = ({
-                  />
+    <>
+      <Row gutter={[16, 16]}>
+        <Col>
+          <Row gutter={[16, 16]}>
+            <Col xs={18}>
+              <Select
+                showArrow
+                bordered={false}
+                className="tw-text-body webhook-filter-select cursor-pointer"
+                mode="multiple"
+                options={statuses}
+                placeholder="Filter by status"
+                style={{ minWidth: '148px' }}
+                onChange={onStatusFilter}
+              />
+            </Col>
+            <Col xs={6}>
+              <Space
+                align="center"
+                className="tw-w-full tw-justify-end"
+                size={16}>
+                {filteredData.length > 0 && (
+                  <Tooltip
+                    placement="left"
+                    title={
+                      addWebhookPermission
+                        ? 'Add Webhook'
+                        : NO_PERMISSION_FOR_ACTION
+                    }>
+                    <Button
+                      className={classNames('tw-h-8 tw-rounded ')}
+                      data-testid="add-webhook-button"
+                      disabled={!addWebhookPermission}
+                      size="small"
+                      theme="primary"
+                      variant="contained"
+                      onClick={onAddWebhook}>
+                      Add {WEBHOOKS_INTEGRATION[webhookType]}
+                    </Button>
+                  </Tooltip>
                 )}
-              </>
-            ) : (
-              fetchErrorPlaceHolder('No webhooks found for applied filters')
-            )}
-          </Col>
-        </Row>
-      </Col>
-      <Col flex="312px">
-        <div className="webhook-right-panel">{rightPanel}</div>
-      </Col>
-    </Row>
+              </Space>
+            </Col>
+            <Col xs={24}>
+              <WebhookTable
+                webhookList={filteredData || []}
+                onDelete={(data) => setWebhook(data)}
+                onEdit={onClickWebhook}
+              />
+              {Boolean(!isNil(paging.after) || !isNil(paging.before)) && (
+                <NextPrevious
+                  currentPage={currentPage}
+                  pageSize={PAGE_SIZE}
+                  paging={paging}
+                  pagingHandler={onPageChange}
+                  totalCount={paging.total}
+                />
+              )}
+            </Col>
+          </Row>
+        </Col>
+      </Row>
+      {selectedWebhook && (
+        <ConfirmationModal
+          bodyText={`You want to delete webhook ${selectedWebhook.name} permanently? This action cannot be reverted.`}
+          cancelText="Cancel"
+          confirmButtonCss="tw-bg-error hover:tw-bg-error focus:tw-bg-error"
+          confirmText="Delete"
+          header="Are you sure?"
+          onCancel={() => setWebhook(undefined)}
+          onConfirm={handleDelete}
+        />
+      )}
```

Comment:
instead of keeping outside of `<Row></Row>` we can include this on same block, it won't affect the layout

Reply:
Sure.